src/ctapipe/tools/process.py

```diff
@@ -187,6 +201,19 @@ def setup(self):
         self.write = self.enter_context(
             DataWriter(event_source=self.event_source, parent=self)
         )
+
+        if (
+            self.calibrate.process_flatfield_events
+            or self.calibrate.process_pedestal_events
+        ):
+            self.log.warning(
```

Comment:
Please don't mess with the configuration system like this.

Reply:
Okay, removed!